lumen/ai/utils.py

```diff
@@ -172,15 +172,20 @@ async def get_schema(
     for field, spec in schema.items():
         if "enum" not in spec:
             continue
-        elif not include_enum:
+
+        limit = get_kwargs.get("limit")
+        if not include_enum:
             spec.pop("enum")
-        elif "limit" in get_kwargs and len(spec["enum"]) > get_kwargs["limit"]:
+        elif limit and len(spec["enum"]) > limit:
```

Comment:
This will error if a user gives a limit as a string as a kwarg. I'm not sure if there is a check for this somewhere else, but we may want to set a default value when we do a `get` from the dictionary.

Reply:
Thus far, ai/utils is used internally so passing a `str` shouldn't be much of an issue. We also set a default value above if the user doesn't input it:
```python
        if "limit" not in get_kwargs:
            get_kwargs["limit"] = 100
```